src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -135,7 +136,7 @@ const DomainList = (props) => {
               {item.name}
             </Text>
 
-            {item.description !== null && (
+            {((!!item.user_id_follower && !!item.description) || isDomain) && (
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering the description has been changed. Ensure that this change doesn't affect other parts of the application where this component is used.
```diff
-            {item.description !== null && (
+            {((!!item.user_id_follower && !!item.description) || isDomain) && (
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->